kube/deploy.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 set -euo pipefail # make bash quit if something weird happens
 
-export KUBE_NAMESPACE=${ENVIRONMENT}
+export KUBE_NAMESPACE=${DRONE_STEP_NAME}
```

Comment:
Does this no open us up to mistakes in the future? Unsure if environment was more explicit.

Reply:
I've never really liked how we used `ENVIRONMENT` only to immediately pass it through into `KUBE_NAMESPACE`. Less code is better when it comes to Bash, I think.

We can't use `$DRONE_STEP_NAME` in the .drone.yml directly as variables are interpolated prior to any pipeline being run (so `$DRONE_STEP_NAME` is unset). That means you can't do
```yaml
- environment:
    ENVIRONMENT: ${DRONE_STEP_NAME} # $ENVIRONMENT is now ""
```

We could do something like
```bash
ENVIROMENT=${DRONE_STEP_NAME}
export KUBE_NAMESPACE=${ENVIRONMENT}
```
if you really wanted to.

Reply:
Could we add some documentation in a README about the step naming convention in the `drone.yml`? I agree in hindsight that setting `ENVIRONMENT` to then just pass to something else.